src/core_plugins/kbn_vislib_vis_types/public/controls/heatmap_options.html

```diff
@@ -109,16 +109,18 @@
         <table class="vis-editor-agg-editor-ranges form-group" ng-show="vis.params.colorsRange.length">
           <tr>
             <th>
-              <label>From</label>
+              <label id="heatmap-custom-range-from">From</label>
```

Comment:
Our current [html styleguide](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md#camel-case-id-and-other-attribute-values) says, we should use camel case for ids - felt unintuitive for me, too :D